docs/config.md

```diff
@@ -273,6 +276,9 @@ profile_models: <comma-delimited-string-list|list|profile_model>
 # List of composing models for BLS models
 bls_composing_models: <comma-delimited-string-list>
 
+# List of an ensemble's composing models that should be profiled using CPU instances only
```

Comment:
BLS as well, right? Not just ensemble? Or do we not have control of that?

Reply:
Yes, this will work for BLS as well. I've changed the assertion to allow this feature for BLS models and updated the documentation.

Reply:
ok but this doc statement still says ensemble